vgrep.go

```diff
@@ -122,30 +122,29 @@ func main() {
 			fmt.Fprintf(os.Stderr, "error loading cache: %v\n", err)
 			os.Exit(1)
 		}
+
+		// Execute the specified command and/or jump directly into the
+		// interactive shell.
+		if len(v.matches) > 0 {
+			v.commandParse()
+			v.waiter.Wait()
+		}
+
+		os.Exit(0)
 	} else {
```

Comment:
Given we always `os.Exit` above, we don't need the else branch anymore. Can you remove it?

Reply:
Absolutely.